overlord/snapstate/snapmgr.go

```diff
@@ -771,6 +773,19 @@ func (m *SnapManager) undoUnlinkCurrentSnap(t *state.Task, _ *tomb.Tomb) error {
 
 	// mark as active again
 	Set(st, ss.Name(), snapst)
+
+	// FIXME: would be good to undo in undoStartSnapServices
```

Comment:
Part of the point of stop was to avoid this. Is it a left over or do I misunderstand?